src/main/java/mate/academy/bookshop/dto/shoppingcart/cart/CartItemResponseDto.java

```diff
@@ -0,0 +1,20 @@
+package mate.academy.bookshop.dto.shoppingcart.cart;
+
+import jakarta.validation.constraints.NotBlank;
+import lombok.Data;
+import mate.academy.bookshop.model.Book;
+import mate.academy.bookshop.model.ShoppingCart;
+
+@Data
+public class CartItemResponseDto {
+    private static final String NOT_BE_NULL = "Can't be null";
```

Comment:
this constants like in other classes are unnecessary. But if you add them, so namings should not be like 
"NOT_BE_BULL" - it is uninformative. For example it can be like "NULL_FIELD_MESSAGE"